### DISCUSSION TITLE
Publii site renders in Firefox, but not Chrome

### DISCUSSION THREAD

**Discussion by MDouglas-22** [GetPublii/Publii#1924] [General] [answered]
Hello, I am creating my first Publii site. [https://mdouglas-22.github.io/resume-publii/]
The themes are not rendering in Chrome. I found a thread in the old discussion forum outlining the same problem, except it worked in Firefox. I checked and my site does work there too. The thread said it has something to do with my site being an https, but the files being http. 
How do I change the Publii generated files to match the https site?

**Comment by candidexmedia** [comment, marked as answer]
Have you solved your issue? It looks like it works on my end.

**Reply by michaelfluegge** [reply]
your site works in chrome on my end, too.

**Reply by MDouglas-22** [reply]
Yes, it seems to be working now. Perhaps I just needed to wait longer for the files to have been uploaded. Thank you both for the responses.
